Update clause_1.adoc (#38)
opengeospatial/geosemantics-dwg: white_paper/clause_1.adoc

== Introduction
//write text in as many clauses as necessary. Use one document or many, your choice!

The spatial aspect of data is a core data dimension. Spatial information provides the capability to answer the question "where", providing additional context to whatever it relates to. The authors of this White Paper believe that semantic and graph technologies, together with  strategies for Linked Data, will increase the value that can be extracted from (geo)spatial data.

> Although many current and past work within the OGC involves semantic and graph technologies, there is one main applicable OGC standard: GeoSPARQL. GeoSPARQL offers both semantics for representing geospatial data and it specifies how geospatial data that are expressed using the Resource Description Framework (RDF) can be queried using SPARQL, the query language for RDF. 

Reviewer: ```suggestion
Although much current and past work within the OGC involves semantic and graph technologies, there is only one applicable OGC standard: GeoSPARQL. GeoSPARQL both offers semantics for representing geospatial data, and specifies how geospatial data that are expressed using the Resource Description Framework (RDF) can be queried using SPARQL, the query language for RDF. 
```

The purpose of this paper is to provoke further thought about a best course for further development of the GeoSPARQL standard, and to invite active involvement in that development. Particularly, the involvement of people and organisations that until now have not been able to put GeoSPARQL to good use, either because of perceived limitations or because of unfamiliarity with the standard, will be highly valued. Also, since one development under consideration is to make provisions for use of GeoSPARQL with non-geographic spatial data, those that see opportunities for using spatial data in a broad sense together with the aforementioned technologies are cordially invited to share their views.  

== Semantic Technologies

> Semantic technologies, in the scope of this document, relate to the use of formal ontologies to describe concepts: the relationships between "things" and categories of "things". There are a number of goals for using these technologies, which often relate to providing the capability to reason over the data and simplified dealing with heterogeneous data sources. Example goals include enabling machine readability, and federated searching. A major area in which semantic technologies are uses is the Semantic Web, the part of the World Wide Web dealing with semantics.

Reviewer 2: ```suggestion
Semantic technologies, in the scope of this document, relate to the use of formal ontologies to describe concepts: the relationships between "things" and categories of "things". There are a number of goals for using these technologies, which often relate to providing the capability to reason over the data and simplified dealing with heterogeneous data sources. Example goals include enabling machine readability, and federated searching. A major area in which semantic technologies are used is the Semantic Web, the part of the World Wide Web dealing with semantics.
```
